CharacterGenerator/CharacterGenerator/Soldier.h

```diff
@@ -0,0 +1,26 @@
+//
+//  Soldier.h
+//  CharacterGenerator
+//
+//  Created by Z on 8/2/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface Soldier : NSObject
+
+@property (nonatomic) NSString *name;
+
+@property (nonatomic) NSString *primaryWeapon;
```

Comment:
Thank you for looking into our project and providing detailed suggestions for us. Thanks a lot, Eric.